pkg/client/clientset/versioned/clientset.go

```diff
@@ -0,0 +1,98 @@
+/*
+Copyright The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Code generated by client-gen. DO NOT EDIT.
+
+package versioned
+
+import (
+	fiaasv1 "github.com/fiaas/fiaas-go-client/pkg/client/clientset/versioned/typed/fiaas.schibsted.io/v1"
+	discovery "k8s.io/client-go/discovery"
+	rest "k8s.io/client-go/rest"
+	flowcontrol "k8s.io/client-go/util/flowcontrol"
+)
+
+type Interface interface {
+	Discovery() discovery.DiscoveryInterface
+	FiaasV1() fiaasv1.FiaasV1Interface
+	// Deprecated: please explicitly pick a version if possible.
+	Fiaas() fiaasv1.FiaasV1Interface
+}
+
+// Clientset contains the clients for groups. Each group has exactly one
+// version included in a Clientset.
+type Clientset struct {
+	*discovery.DiscoveryClient
+	fiaasV1 *fiaasv1.FiaasV1Client
+}
+
+// FiaasV1 retrieves the FiaasV1Client
+func (c *Clientset) FiaasV1() fiaasv1.FiaasV1Interface {
+	return c.fiaasV1
+}
+
+// Deprecated: Fiaas retrieves the default version of FiaasClient.
```

Comment:
It seems strange to generate deprecated functions in a new library. Is this an artifact of the codegenerator, or can we disable it?